.github/workflows/ci_tests.yaml

```diff
@@ -52,7 +52,7 @@ jobs:
             optional-packages: ''
           - python-version: 3.9
             numpy-version: '1.20'
-            optional-packages: ''  # 'geopandas'
+            optional-packages: 'geopandas'
```

Comment:
Seems like `geopandas` on conda-forge split into two packages in https://github.com/conda-forge/geopandas-feedstock/pull/90. There is `geopandas-base` (light version) and `geopandas` (full version). An option might be to install `geopandas-base` and `fiona` (needed for doing I/O, c.f. #1000) instead of the full `geopandas` package (which brings in stuff like `rtree` and `mapclassify` that we probably don't need). Or we could keep things as is and just install `geopandas` since that's what most users would do.